assessment/scripts/2_compute_metrics.py

```
@@ -0,0 +1,467 @@
+                    # Check AF vs GT concordance
+                    if gt_dosage == 0 and af_val < 0.1:
+                        af_gt_concordant += 1
+                    elif gt_dosage == 1 and 0.3 <= af_val <= 0.7:
+                        af_gt_concordant += 1
+                    elif gt_dosage == 2 and af_val > 0.9:
+                        af_gt_concordant += 1
+                    else:
+                        af_gt_discordant += 1
+                else:
+                    missing_af += 1
+        except:
```

Ruff (0.15.5)

[error] Do not use bare `except`

(E722)